packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -273,7 +283,7 @@
             top: 0;
 
             &--disable {
-                cursor: not-allowed;
+                cursor: not-allowed !important;
```

Comment:
whyyyyyyyyyyyyy

Reply:
I had to add `!important` because it would not override the cursor pointer set in popoverIcons, and before I added the mixin, I had saw the previous CSS would not override the cursor pointer as well